### PR TITLE
aurman: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/aurman.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+> An Arch Linux utility to build and install packages from the Arch User Repository.
+> See also `pacman`.
+> More information: <https://github.com/polygamma/aurman>.
+
+- Synchronize and update all packages:
+
+`aurman --sync --refresh --sysupgrade`
+
+- Synchronize and update all packages without show changes of PKGBUILD files:
```
Comment (bl-ue) [inline pages/linux/aurman.md:11]:
```suggestion
- Synchronize and update all packages without show changes of `PKGBUILD` files:
```

File: pages/linux/aurman.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`aurman --sync --refresh --sysupgrade --noedit`
+
+- Install a new package:
+
+`aurman --sync {{package_name}}`
+
+- Install a new package without show changes of PKGBUILD files:
```
Comment (bl-ue) [inline pages/linux/aurman.md:19]:
```suggestion
- Install a new package without show changes of `PKGBUILD` files:
```

**Review by bl-ue — COMMENTED** [review]
Don't you think we should backtick 'PKGBUILD'?

**Review by bl-ue — APPROVED** [review]

File: pages/linux/aurman.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- Search the package database for a keyword from the official repositories and AUR:
+
+`aurman --sync --search {{keyword}}`
+
+- Remove a package and its dependencies:
+
+`aurman --remove --recursive --nosave {{package_name}}`
+
+- Remove not installed packages and unused repositories from the cache (use two `--clean` flags to clean all packages):
```
Comment (sbrl) [inline pages/linux/aurman.md:35]: `Remove not installed packages` doesn't sounds very intuitive at first here. Is there another way we could phrase this?

**Review by sbrl — APPROVED** [review]
Just a small suggestion :-)

File: pages/linux/aurman.md (hunk 3 above)
Comment (marchersimon) [reply pages/linux/aurman.md:35]: @navarroaxel How about `Remove packages that are no longer installed from the cache as well as currently unused sync databases`? That's what the page referred to in `aurman --help` says.

Comment (navarroaxel) [reply pages/linux/aurman.md:35]: Close, but not exactly the AUR packages are git repositories with a `PKGBUILD` file to build your own package. So `aurman` keeps some builds and that cloned Git repos in `.cache/aurman/`, so this command:
- Remove AUR cloned Git repositories from `aurman` cache for packages that are no longer installed.
- Remove downloaded packages that are no longer installed from the `pacman` cache.
- Also, the unused sync databases are removed but is not important as the previous ones.

Comment (bl-ue) [reply pages/linux/aurman.md:35]: What about something else, like `Clear the package cache`?

Comment (navarroaxel) [reply pages/linux/aurman.md:35]: Why not, is a more general description but we can explain every step of a command.

Comment (sbrl) [reply pages/linux/aurman.md:35]: Sounds good!

**Review by marchersimon — APPROVED** [review]

**Pull request merged**